mindsdb/api/mongo/op_msg_responders/add_shard.py

```diff
@@ -0,0 +1,40 @@
+from mindsdb.api.mongo.classes import Responder
+import mindsdb.api.mongo.functions as helpers
+
+# GET OpMSG=OrderedDict([('_addShard', 1), ('shardIdentity', OrderedDict([('shardName', 'shard0000'), ('clusterId', ObjectId('5f6c7d3e4bf2f9c3da4dc92f')), ('configsvrConnectionString', 'replconf/127.0.0.1:27000')])), ('$db', 'admin')])
+# real full response for v4.4 = {
+#     "ok": 1,
+#     "$gleStats": {
+#         "lastOpTime": {
+#             "ts": 6876008835761307649,
+#             "t": 1
+#         }
+#         "electionId": ObjectId(7f:ff:ff:ff:00:00:00:00:00:00:00:01)
+#     },
+#     "lastCommittedOpTime": 6876008835761307649,
+#     "$configServerState": {
+#         "opTime": {
+#             "ts": 6876008831466340353,
+#             "t": 1
+#         }
+#     },
+#     "$clusterTime": {
+#         "clusterTime": 6876008835761307650,
+#         "signature": {
+#             "hash": 00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00,
+#             "keyId": 0
+#         }
+#     },
+#     "operationTime": 6876008835761307649
+# }
```

Comment:
Is it worth it to keep these large bits of logic commented out ?

Reply:
yes, i think we should keep it, for a time at least. Im not sure all problems with v4.4 gone. That not easy get dump of real answer. So if we will have problems with this answer - will be much easier find reason.